Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -230,7 +233,6 @@ Stratify by covariates of interest to explore whether any key covariates impact
 
 One of the key questions when looking at PD markers is to determine if there is a dose-response relationship, and if there is, what dose is necessary to achieve the desired effect? Simple dose-response plots can give insight into these questions.
 
-
 ### PD marker by Dose, for endpoint of interest, mean (95% CI) by Dose
```

Comment:
Did you consider having the xticks be at the actual dose levels, instead of evenly spaced?  I guess they'd be hard to read when the doses are close together, so perhaps in those cases, only some doses would be listed.  Alternatively, could rotate the orientation of the text.